### DISCUSSION TITLE
Issue to use IBM Datastage Hierachical Stage Harman plug-in installed.

### DISCUSSION THREAD

**Discussion by dajas** [airsdk/Adobe-Runtime-Support#2128] [Q&A]
Hello,  

We have installed the HARMAN plug-in ("AIR runtime including WebKit - version 33.1.1.744") on our machine to use some of the features of the IBM Infosphere Information Suite as mentioned on this IBM support page

 https://www.ibm.com/support/pages/hierarchical-data-stage-flash-player-end-support-dec-2020 .  

As per the documentation, it is asking to contact HARMAN on how to do the URL filtering.  How can we do that? Where do I locate this file “XMLPackController.swf”  so that I can make the mentioned changes?

Thanks,
dajas

**Comment by sramaraju429** [comment]
hi

I am facing similar issue. Are you able to fix this issue?
Also from where did you install HARMAN plug-in ("AIR runtime including WebKit - version 33.1.1.744")  ? Is there any license cost ?
Please help with these details.

Thanks

**Reply by ajwfrost** [reply]
Hi

The AIR runtime (from https://airsdk.harman.com/runtime - scroll down for the .744 build, available under a free EULA) is not going to help with the IBM Data Stage app. Instead you'd need the Flash Player itself, which we still support for enterprise customers, and yes there is a license fee for this solution.

Feel free to contact us via adobe.support at harman.com and mention that you need the Flash Player for IBM DataStage, someone will get back to you with the license agreement details.

thanks
